File: SpringBootApp/src/main/java/com/example/springbootapp/data/dto/CustomerProfileDto.java
package com.example.springbootapp.data.dto;

import com.example.springbootapp.data.entities.Enums.Gender;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class CustomerProfileDto {
    private String id;
    private String username;
    private String email;
    private String firstName;
    private String lastName;
    private String birthDate;
    private Gender gender;
    private String favouriteTeam;
}

File: ClientAdmin/app/src/main/java/com/example/clientadmin/model/dto/CustomerProfileDto.kt
package com.example.clientadmin.model.dto

import com.example.clientadmin.model.enumerator.Gender
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class CustomerProfileDto (
    val id : String,
    val username: String,
    val email: String,
    val firstName: String,
    val lastName: String,
    val birthDate: String,
    val gender: Gender,
    val favouriteTeam: String,
)